pkg/util/nsenter/nsenter.go

```diff
@@ -122,3 +124,19 @@ func (ne *Nsenter) SupportsSystemd() (string, bool) {
 	systemdRunPath, hasSystemd := ne.paths["systemd-run"]
 	return systemdRunPath, hasSystemd
 }
+
+// KubeletPath returns the path name on the host after the evaluation of
```

Comment:
How about something like "KubeletPath returns the path name that can be accessed by containerized kubelet, after evaluating symlinks on the host."

Reply:
Updated it in https://github.com/kubernetes/kubernetes/pull/62903/commits/f611508a9a2569460a3e9db86448c1a7f532c16c. 
It actually returns two path names, I updated the second part, any other suggestions?